projects/Generic/linux/linux.x86_64.conf

```diff
@@ -3147,7 +3147,7 @@ CONFIG_SND_DYNAMIC_MINORS=y
 CONFIG_SND_MAX_CARDS=32
 # CONFIG_SND_SUPPORT_OLD_API is not set
 CONFIG_SND_PROC_FS=y
-# CONFIG_SND_VERBOSE_PROCFS is not set
+CONFIG_SND_VERBOSE_PROCFS
```

Comment:
I can add it tomorrow also.
For the other platforms, I have no idea if it's important for the
snd-hda-intel module since I don't think it's included in non x86 platform
. But the verbose sound fs can be useful in those too.
On Feb 7, 2016 18:37, "MilhouseVH" <notifications@github.com> wrote:

> In projects/Generic/linux/linux.x86_64.conf
> <https://github.com/OpenELEC/OpenELEC.tv/pull/4684#discussion_r52127319>:
>
> > @@ -3147,7 +3147,7 @@ CONFIG_SND_DYNAMIC_MINORS=y
> >  CONFIG_SND_MAX_CARDS=32
> >  # CONFIG_SND_SUPPORT_OLD_API is not set
> >  CONFIG_SND_PROC_FS=y
> > -# CONFIG_SND_VERBOSE_PROCFS is not set
> > +CONFIG_SND_VERBOSE_PROCFS
>
> +CONFIG_SND_VERBOSE_PROCFS
>
> Shouldn't this be =y or something?
>
> And what about other platforms (eg. RPi etc.) - is this option relevant
> there too? Aren't we still maintaining Nvidia_Legacy for the time being?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenELEC/OpenELEC.tv/pull/4684/files#r52127319>.
>